requirements/base.txt

```diff
@@ -1,32 +1,32 @@
 # Django itself and its requirements (pytz)
 Django==4.2.11
-pytz==2023.3
+pytz==2024.1
 
 # Asynchronous tasks
 celery[redis]==5.3.1
 
 # Static files
-whitenoise==6.0.0
+whitenoise==6.6.0
 
 # Handling and processing uploaded images
-Pillow==10.2.0
+Pillow==10.3.0
 
 # Our API
-djangorestframework==3.14.0
+djangorestframework==3.15.1
 djangorestframework-jsonp==1.0.2
 
 # Django helpers
-django-extensions==3.1.5
+django-extensions==3.2.3
 
 # Form helpers
-crispy-bootstrap4==2022.1
-django-crispy-forms==2.0
+crispy-bootstrap4==2024.1
+django-crispy-forms==2.2
 
 # Authentication
-django-allauth==0.49.0
+django-allauth==0.63.3
```

Comment:
I know we hit some various issues with upgrades here on RTD, but we're doing way more complex stuff there, so hopefully it'll just work.